docs/integrations/sources/facebook-pages-migrations.md

```diff
@@ -0,0 +1,40 @@
+# Facebook Pages Migration Guide
+
+## Upgrading to 1.0.0
+
+:::note
+This change is only breaking if you are syncing stream `page`.
```

Comment:
Page

Reply:
we need to tell them what fields are disappearing. or if any of them are being replaced with a different name

Reply:
Fixed